tools/config/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "applicationinsights-web-config",
-    "description": " Application Insights JavaScript SDK - Web Config",
```

Comment:
actually, I created the config folder a long time ago with build task defined and release scripts. so moved the CDN to that folder and changed the version from 0.1.0 to 1.0.0